lib/mayaUsd/commands/Readme.md

```diff
@@ -0,0 +1,440 @@
+# Common Plug-in Commands
+
+This read-me documents the common commands that are shared by the various
+USD plug-ins made by Autodesk, Pixar and Animal Logic.
+
+## Common Plug-in Contents
+
+The common parts of Maya USD plug-in provides base commands to import and
+export USD files from scripts. The name of the command as seen by the
+end-user will vary for each specific plug-in. In the following table,
+we give an example of what the final command name would be in Maya,
+in this instance for the mayaUsd plug-in.
+
+| Class Name                     | Example Real Cmd Name    | Purpose                                |
+| ------------------------------ | ------------------------ | -------------------------------------- |
+| MayaUSDImportCommand           | mayaUSDimport            | Command to import from a USD file      |
+| MayaUSDExportCommand           | mayaUSDExport            | Command to export into a USD file      |
+| MayaUSDListShadingModesCommand | mayaUSDListShadingModes  | Command to get available shading modes |
+| EditTargetCommand              | mayaUsdEditTarget        | Command to set or get the edit target  |
+| LayerEditorCommand             | mayaUsdLayerEditor       | Manipulate layers                      |
+| LayerEditorWindowCommand       | mayaUsdLayerEditorWindow | Open or manipulate the layer window    |
+
+Each base command class is documented in the following sections.
+
+
+## `MayaUSDImportCommand`
+
+### Command Flags
+
+| Long flag              | Short flag | Type           | Default                           | Description |
+| ---------------------- | ---------- | -------------- | --------------------------------- | ----------- |
+| `-apiSchema`           | `-api`     | string (multi) | none                              | Imports the given API schemas' attributes as Maya custom attributes. This only recognizes API schemas that have been applied to prims on the stage. The attributes will properly round-trip if you re-export back to USD. |
+| `-assemblyRep`         | `-ar`      | string         | `Collapsed`                       | If the import results in the creation of assembly nodes, this value specifies the assembly representation that will be activated after creation. If empty, no representation will be activated. Valid values are: `Collapsed`, `Expanded`, `Full`, `Import`, `(empty)`. `Import`: No USD reference assembly nodes will be created, and the geometry will be imported directly. `(empty)`: The assembly is created but is left unloaded after creation. See "Importing as Assemblies" below for more detail on assembly node creation. |
```

Comment:
"Assembly" here refers to Maya's `MFnAssembly` which is only relevant for the `pxrUsd` plugin, so I would assume we probably want to remove `-assemblyRep` here.

Reply:
Yes, it is planned for another PR.

Reply:
The reason being that assembly-related command flags are still on the base class, probably by mistake we think, so it will require a code refactor.

Reply:
Finally I did the rework to move the assemblyRep flag in this PR.